### PR TITLE
Add WalletLink connection

### PR DESCRIPTION
**Pull request by t0mcr8se**

### CODE REVIEW

**Review by leonprou — APPROVED** [review]

File: src/connectors/index.ts [hunk 1]
```diff
@@ -89,7 +89,7 @@ export const portis = new PortisConnector({
 // mainnet only
 export const walletlink = new WalletLinkConnector({
   url: ETHEREUM_NETWORK_URL,
```
Comment (mul53) [inline src/connectors/index.ts:91]: @leonprou is this integration for mainnet or fuse?

Comment (t0mcr8se) [reply src/connectors/index.ts:91]: @leonprou 
@mul53 
This is an old comment, not sure what it means, tested the connection on fuse and it worked fine, also tried to switch networks from the coinbase wallet and that reflects on the front-end

Comment (t0mcr8se) [reply src/connectors/index.ts:91]: Basically I think it uses the rpc provider from the coinbase wallet, not from `url: ETHEREUM_NETWORK_URL`

Comment (mul53) [reply src/connectors/index.ts:91]: @t0mcr8se great!, let me just do some final checks before we merge it

**Comment by leonprou** [comment]
@mul53 Can we merge this?

**Comment by mul53** [comment]
@leonprou checked it's working using the coinbase mobile wallet, though through the browser extension it didn't allow specifying the network, maybe i missed something. The mobile wallet works well so should be good enough. @t0mcr8se just fix the lint issues then i will approve

**Comment by t0mcr8se** [comment]
@mul53 Yeah just tested with the extension and looks like there's a bug with switching the network, however, it switches successfully when you refresh fusefi after changing the network from the wallet

**Review by mul53 — APPROVED** [review]
@t0mcr8se great work, looks good! @leonprou good to go

**Pull request merged**